src/alsa.c

```diff
@@ -476,10 +475,11 @@ static int convert_prange(long val, long min, long max) {
  * Adjusts the current volume and sends a notification (if enabled).
  *
  * @param vol new volume value
+ * @param dir select direction
```

Comment:
This should also say the valid input... afais `0` is "no direction, `1` is "up" and `-1` is down? It's also debatable if we really need a full `int` for this, but then again that's often how people do C.